travis/wptTests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -ev
+if [ "${WPTKEY}" != "false" ]; then
```

Comment:
What scenario is this meant to cover? Is it perhaps "$TRAVIS_PULL_REQUEST" != "false", as described by
https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions ?